cmd/transporter/javascript_builder.go

```diff
@@ -295,14 +301,35 @@ func (js *JavascriptBuilder) Build() error {
 
 // Run runs each of the transporter pipelines sequentially
 func (js *JavascriptBuilder) Run() error {
+	var g group.Group
 	for _, p := range js.pipelines {
-		err := p.Run()
-		if err != nil {
-			return err
-		}
+		g.Add(func() error {
+			return p.Run()
+		}, func(error) {
+			p.Stop()
```

Comment:
alright, well this is okay i think. we can reconsider when we start finding other use acses for this